plugins/optimization-detective/storage/data.php

```diff
@@ -141,42 +141,45 @@ function od_get_url_metrics_slug( array $query_vars ): string {
 }
 
 /**
- * Computes nonce for storing URL Metrics for a specific slug.
+ * Computes HMAC for storing URL Metrics for a specific slug.
  *
  * This is used in the REST API to authenticate the storage of new URL Metrics from a given URL.
  *
  * @since 0.1.0
+ * @since n.e.x.t Renamed from od_get_url_metrics_storage_nonce().
```

Comment:
Better to consider this an entirely new function. I would argue it wasn't "just" renamed, the purpose was also changed as it now returns a different kind of value.

Reply:
Sure. Done in 9d6e80784